File: Parth Traders.Domain/Entity/OrderDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Parth_Traders.Domain.Entity
{
    public class OrderDetails
    {
        public long OrderDetailId { get; set; }

        public virtual Order OrdersData { get; set; }

        public virtual Product ProductsData { get; set; }

        public long Price { get; set; }

        public long Quantity { get; set; }

        public long Discount { get; set; }

        public long Total { get; set; }

        public DateTime BillDate { get; set; }
    }
}

File: Frontend/parth-traders/src/app/admin/shared/order-details.ts
import { Product } from '../product/product';
import { Order } from './order';

export interface OrderDetails {
  position: number;
  order: Order;
  product: Product;
  pricePerPiece: number;
  quantityPurchased: number;
  discount: number;
  total: number;
  billDate: Date;
}
